CITATION.cff

```diff
@@ -0,0 +1,49 @@
+cff-version: 1.2.0
+message: "If you use this software, please cite it as below."
+authors:
+- family-names: "Pocock"
+  given-names: "Johnathan"
+  orcid: "https://orcid.org/0000-0002-2866-981X"
+- family-names: "Graham"
+  given-names: "Simon"
+  orcid: "https://orcid.org/0000-0002-2214-8212"
+- family-names: "Vu"
+  given-names: "Quoc Dang"
+  orcid: "https://orcid.org/0000-0003-2075-9201"
+- family-names: "Jahanifar"
+  given-names: "Mostafa"
+  orcid: "https://orcid.org/0000-0001-5842-0460"
+- family-names: "Deshpande"
+  given-names: "Srijay"
+  orcid: "https://orcid.org/0000-0000-0000-0000"
+- family-names: "Hadjigeorghiou"
+  given-names: "Giorgios"
+  orcid: "https://orcid.org/0000-0003-2734-8915"
+- family-names: "Shephard"
+  given-names: "Adam"
+  orcid: "https://orcid.org/0000-0003-0969-2990"
+- family-names: "Bashir"
+  given-names: "Raja Muhammad Saad"
+  orcid: "https://orcid.org/0000-0000-0000-0000"
+- family-names: "Bilal"
+  given-names: "Mohsin"
+  orcid: "https://orcid.org/0000-0001-8632-2729"
+- family-names: "Lu"
+  given-names: "Wenqi"
+  orcid: "https://orcid.org/0000-0002-7838-0918"
+- family-names: "Epstein"
+  given-names: "David"
+  orcid: "https://orcid.org/0000-0001-8634-7168"
+- family-names: "Minhas"
+  given-names: "Fayyaz"
+  orcid: "https://orcid.org/0000-0001-9129-1189"
+- family-names: "Rajpoot"
+  given-names: "Nasir M."
+  orcid: "https://orcid.org/0000-0002-4706-1308"
+- family-names: "Raza"
+  given-names: "Shan E Ahmed"
```

Comment:
Not sure if it matters, but some initials have a period after them and some do not.